notification-support/src/main/resources/avro/stream-registry-notification.avdl

```diff
@@ -0,0 +1,46 @@
+@namespace("com.expediagroup.streamplatform.streamregistry.avro")
+protocol StreamRegistry {
+
+  enum AvroKeyType {
+    DOMAIN, SCHEMA, STREAM, STREAM_VERSION
+  }
+
+  record Tag {
+    string name;
+    string value;
+  }
+
+  record AvroKey {
+    string id;
+    union {null, string} domain = null;
```

Comment:
Do we want the domain here? Wouldn't that be in the parent for schema and stream? When we enable this for the physical entities (zone, infra, binding etc) these won't have domain.

Reply:
Got it, I basically used the initial avro-schema with little modifications and I considered it to be used in domain/stream/schema-like events. I'll modify it.

Reply:
Domain removed from key